docs/snippets/vue/list-story-with-sub-components.ts-4-9.mdx

```diff
@@ -0,0 +1,41 @@
+```ts
+// List.stories.ts
+
+// Replace vue3 with vue if you are using Storybook for Vue 2
```

Comment:
Remove this comment as we're no longer providing support for Vue 2 and if this is not going to be backported to 7.x, we should mention it. Apply it in the other Vue examples used in this pull request, if you don't mind.